staging/sales_index.json

```diff
@@ -27,6 +27,12 @@
    {
       "chainId": 3,
       "saleManagerAddress": "0xb2a2934de53c1dd6c330ee1db8ccb138a64837db",
+      "saleId": "0x52d12379e10ea4ba64a4643eaad160a5aad1838bad6a10d9bcf9ac23a439ade6",
+      "saleName": "nikenikitos9(NIKE)",
+      "logo": "https://wampi.ru/image/RuaSyVV",
+      "favicon": "https://wampi.ru/image/Rual5Oz",
+      "projectWebsite": "https://twitter.com/Nikita08794843",
+      "saleDescription": "SALE example tokens nikenikitos9 for test",
```

Comment:
Could you start from the beginning with your changes? It looks like you've merged two sets of sale data into one structure.

Reply:
yes okey)